func/warble.cc

```diff
@@ -88,13 +128,15 @@ std::string WarbleImpl::CreateWarble(const WarbleRequest* request, WarbleReply*
   std::string text = request->text();
   std::string parent_id = request->parent_id();
 
+  KvstoreClient client(grpc::CreateChannel("localhost:50001", grpc::InsecureChannelCredentials()));
+
   // Feature Logic
   std::string hash_str = username + "-" + text + "-" + parent_id;
   std::size_t id = std::hash<std::string>{}(hash_str);
   std::string key = "warble-" + id;
   Warble warble = Create_Warble(username,text,std::to_string(id),parent_id);
```

Comment:
You may want to rename one of the two Create Warble functions to give more clarity to what each is doing.